ActindoWorkFlow/UserInterface/08_CoreActions.md

```diff
@@ -4,15 +4,18 @@
 
 ![Core actions](../../Assets/Screenshots/ActindoWorkFlow/Workflows/CoreActions/CoreActions.png "[Core actions]")
 
-To create a workflow, you must include one or several workflow actions.   
-In the *Workflows* module, a certain number of workflow actions is always preconfigured, the so-called *Core actions*. These core actions are used to build control flow structures in a workflow.   
-Other workflow actions are available depending on the plugins installed in the current system.   
+To create a workflow, you must include one or several process actions.   
```

Comment:
Reintheoretisch sollte es "To create a workflow, you must include one or several transitions" heißen, oder? Process action heißt es, wenn es zu den tatsächlich Prozessausführung kommt (so ist es vorher beschrieben, denke ich). Hier ist die Terminologie (transition vs. action) unglücklich.